test/e2e/storage/testsuites/provisioning.go

```diff
@@ -440,6 +443,231 @@ func (p *provisioningTestSuite) DefineTests(driver storageframework.TestDriver,
 		l.testCase.TestDynamicProvisioning(ctx)
 	})
 
+	ginkgo.It("should provision storage with pvc data source for readonly filesystem", func(ctx context.Context) {
+		if !dInfo.Capabilities[storageframework.CapPVCDataSource] {
+			e2eskipper.Skipf("Driver %q does not support cloning - skipping", dInfo.Name)
+		}
+
+		init(ctx)
+
+		if l.config.ClientNodeSelection.Name == "" {
+			// Schedule all pods to the same topology segment (e.g. a cloud availability zone), some
```

Comment:
I believe https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1150 should fix the topology issue with cloning.

Reply:
Good to know. I'll file an internal bug to clean this up after this goes in, as I don't think we should block and wait for another PR to go in (the last one took months)